epoch.go

```diff
@@ -930,19 +930,28 @@ func (e *Epoch) rebroadcastPastFinalizations() error {
 }
 
 func (e *Epoch) indexFinalizationCertificates(startRound uint64) {
-	r := startRound
-	round, exists := e.rounds[r]
-	if !exists {
-		e.Logger.Debug("Round not found", zap.Uint64("round", r))
-		return
-	}
-	if round.fCert.Finalization.Seq != e.Storage.Height() {
-		e.Logger.Debug("Finalization certificate does not correspond to the next sequence to commit",
-			zap.Uint64("seq", round.fCert.Finalization.Seq), zap.Uint64("height", e.Storage.Height()))
-		return
+	maxRound := uint64(0)
```

Comment:
I just noticed that the title of the PR is not aligned with the bug fix, what do you think?